src/Nowruz/modules/settings/components/account/index.tsx

```diff
@@ -0,0 +1,137 @@
+import { AnyAction } from '@reduxjs/toolkit';
+import { useState } from 'react';
+import { Button } from 'src/Nowruz/modules/general/components/Button';
+import { FeaturedIcon } from 'src/Nowruz/modules/general/components/featuredIcon-new';
+import { Input } from 'src/Nowruz/modules/general/components/input/input';
+import { Modal } from 'src/Nowruz/modules/general/components/modal';
+import { SearchDropdown } from 'src/Nowruz/modules/general/components/SearchDropdown';
+import { deleteAccount } from 'src/pages/delete-profile/delete-profile.service';
```

Comment:
deleteAccount is imported from v2 pages, we have removed all v2 pages and here we have a conflict for merging. Please write the logic of each page separately in its own folder, just import APIs from src/core/api